sdk/resourcemanager/azure-resourcemanager-compute/src/main/java/com/azure/resourcemanager/compute/implementation/VirtualMachineImpl.java

```diff
@@ -1993,6 +1994,14 @@ public String userData() {
         return this.innerModel().userData();
     }
 
+    @Override
+    public String capacityReservationGroupId() {
+        if (this.innerModel().capacityReservation() != null) {
+            return this.innerModel().capacityReservation().capacityReservationGroup().id();
```

Comment:
also check `this.innerModel().capacityReservation().capacityReservationGroup()`?

Reply:
Fixed in the [commit#81dafd1](https://github.com/Azure/azure-sdk-for-java/pull/42914/commits/81dafd1492a3812074472ab4583c1e13e86b0a76)